nemoguardrails/library/hallucination/anchor/actions.py

```
@@ -0,0 +1,52 @@
+                "context": source_context,
+                "threshold": threshold
+            },
+            timeout=5
+        )
+        
+        if response.status_code == 200:
+            result = response.json()
+            # If 'allow' is False, drift was detected
+            return result.get("allow", True)
+            
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)